code/lib/cli/src/automigrate/fixes/vta.ts

```diff
@@ -0,0 +1,60 @@
+import { dedent } from 'ts-dedent';
+import { updateMainConfig } from '../helpers/mainConfigFile';
+import type { Fix } from '../types';
+import { getStorybookVersionSpecifier } from '../../helpers';
+
+const logger = console;
+
+interface Options {}
+
+/**
+ */
+export const vta: Fix<Options> = {
+  id: 'visual-testing-addon',
+
+  versionRange: ['<8.0.7', '>=8.0.7'],
+
+  async check({ mainConfig }) {
+    const hadAddonInstalled = !!mainConfig?.addons?.find((addon) =>
+      typeof addon === 'string'
+        ? addon.includes('@chromatic-com/storybook')
+        : addon.name.includes('@chromatic-com/storybook')
+    );
+
+    // @ts-expect-error (user might be upgrading from an older version that still had it)
+    const usesMDX1 = mainConfig?.features?.legacyMdx1 === true || false;
+    const skip = hadAddonInstalled;
+
+    if (skip) {
+      return null;
+    }
+
+    return {};
+  },
+
+  prompt() {
+    return dedent`
+      We've detected that you're not yet using the Visual Testing Addon. Would you like to add it?
+    `;
+  },
+
+  async run({ packageManager, dryRun, mainConfigPath, skipInstall }) {
+    if (!dryRun) {
+      const packageJson = await packageManager.retrievePackageJson();
+      const versionToInstall = getStorybookVersionSpecifier(
+        await packageManager.retrievePackageJson()
+      );
+      await packageManager.addDependencies(
+        { installAsDevDependencies: true, skipInstall, packageJson },
+        [`@storybook/addon-mdx-gfm@${versionToInstall}`]
+      );
```

Comment:
1. Wrong addon
2. The addon doesn't follow SB's versioning so we can't use that for the version, I guess just install latest?

```suggestion
      await packageManager.addDependencies(
        { installAsDevDependencies: true, skipInstall, packageJson },
        ['@chromatic-com/storybook']
      );
```